kubeportal/management/commands/ensure_root.py

```diff
@@ -1,16 +1,33 @@
 from django.core.management.base import BaseCommand
 from django.contrib.auth import get_user_model
 from django.contrib.auth.hashers import make_password
+from os import environ
 
 
 class Command(BaseCommand):
     ''' Make sure that superuser exists. '''
 
     def handle(self, *args, **option):
+        # create django user
         User = get_user_model()
         user, created = User.objects.get_or_create(
             username='root', is_superuser=True, is_staff=True)
-        pw = User.objects.make_random_password()
+
+        '''
+        hard code password when running in development mode
+        use random password as default in case something goes wrong
+        if no root password is set as an environment variable, use 'rootpw'
+        '''
+        pw = None
+        if environ['DJANGO_CONFIGURATION'] == 'Development':
+            print(environ.get('DEV_ROOT_PASSWORD'))
```

Comment:
What is DEV_ROOT_PASSWORD?

Reply:
Woops, I may have left in a print statement from testing. I renamed the variable later. It can be removed.